subcommand/server-acl-init/policies.go

```diff
@@ -0,0 +1,155 @@
+package serveraclinit
+
+import (
+	"bytes"
+	"strings"
+	"text/template"
+)
+
+type rulesData struct {
+	EnableNamespaces               bool
+	ConsulSyncDestinationNamespace string
+	EnableSyncK8SNSMirroring       bool
+	SyncK8SNSMirroringPrefix       string
+}
+
+const snapshotAgentRules = `acl = "write"
+key "consul-snapshot/lock" {
+   policy = "write"
+}
+session_prefix "" {
+   policy = "write"
+}
+service "consul-snapshot" {
+   policy = "write"
+}`
+
+const entLicenseRules = `operator = "write"`
+
+func (c *Command) agentRules() (string, error) {
+	agentRulesTpl := `
+  node_prefix "" {
```

Comment:
The different levels of indentation do not affect the functionality because it doesn't matter to the hcl parser being used (see example output below). I created the indentation so that the OSS use case would be consistent. The UI seems to treat it differently, but there isn't a good way to have the indentation perfect for both namespaces and OSS.

ACL parsing with mixed whitespace: